src/sentry_clj/ring.clj

```diff
@@ -39,6 +44,60 @@
       (response/content-type "text/html")
       (response/status 500)))
 
+(defn- extract-transaction-name
```

Comment:
Hi,

The style that I prefer is to use meta data for privacy, rather than `defn-`. Would you consider changing the `defn-`'s to use `^:private` instead please?

Reply:
All right. I express private function by  using `^:private`.